run/test/specs/locators/browsers.ts

```diff
@@ -0,0 +1,64 @@
+import { LocatorsInterface } from ".";
+
+// SHARED LOCATORS
+export class URLInputField extends LocatorsInterface {
+    public build() {
+        switch (this.platform) {
+            case 'android': 
+                return {
+                    strategy: 'id', 
+                    selector: 'com.android.chrome:id/url_bar'
+                } as const;
+            case 'ios': 
+                return {
+                    strategy: 'accessibility id', 
+                    selector: 'URL'
+                } as const;
+        }
+    }
+}
+
+// ANDROID ONLY 
+export class ChromeUseWithoutAnAccount extends LocatorsInterface {
+    public build() {
+        switch (this.platform) {
+            case 'android': 
+                return {
+                    strategy: 'id', 
+                    selector: 'com.android.chrome:id/signin_fre_dismiss_button',
+                    maxWait: 2000,
```

Comment:
I think this might cause issues down the line, you may want to reuse this somewhere else and have a different maxwait. But I think for now, this is fine